bin/lib/deploy_utils

```diff
@@ -75,9 +75,19 @@ generate_manifest() {
     manifest=$(echo "${manifest}" | bosh-cli int - --vars-file="${bosh_environment}/${deployment_name}-vars.yml")
   fi
 
-  manifest=$(echo "${manifest}" | bosh-cli int - \
-      --vars-file="${bosh_environment}/director.yml" \
-      --var=deployment_name="${deployment_name}")
+  if bosh-cli int "${bosh_environment}/director.yml" --path='/http_proxy'; then
```

Comment:
Based on the way we pipe the manifest contents around and due to the number of variables we need to pass, I don't think this will work out very cleanly. Did you have something in mind?